src/index.ts

```diff
@@ -8,6 +8,7 @@ const getOptionalInput = (name: string) => core.getInput(name) || undefined;
 
 (async () => {
   let githubToken = process.env.GITHUB_TOKEN;
+  let skipCreateGithubRelease = Boolean(process.env.SKIP_CREATE_GITHUB_RELEASE);
```

Comment:
i think that the input option for the action would make more sense and I would call it `createGithubReleases`. It's default value would be `true` so you could disable it with `false`.

Reply:
That's a fair point, I also figure this is a bit specific for this use case so polluting function param list with this boolean flag is not great, hence I wrapped it in `options`. If you think that's overkill, let me know.